Coincap/README.md

```diff
@@ -0,0 +1,3 @@
+# Coincap
+
+This Scripted Connector fetches cryptocurrency data from CoinCap API, converts certain string values to floats, and then sends the modified data to Data API endpoint. 
```

Comment:
Do we need to add some URLs to indicate where we get the data.  For example https://docs.coincap.io/ and